DataCollector/RulerDataCollector.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace Hoathis\Bundle\RulerBundle\DataCollector;
+
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\DataCollector\DataCollectorInterface;
+
+use Hoathis\Bundle\RulerBundle\Ruler\LoggedRuler;
```

Comment:
@Hywan @K-Phoen I think we should update the naming in the bench bundle. I like what we did on console bridge and bundle and find their naming clear:

* what it targets? Symfony
* what it is? A bridge or a bundle
* what it wraps? Console

everything is in the name.